documentation/event-timeline.md

```diff
@@ -0,0 +1,29 @@
+# Pre Event Timeline
+
+1. [Pre Event Support Eng Same Page](pre-event-support-eng-same-page.md)
+
+# Event Timeline
+
+1. Pre-requisite stories
+	- Bug fixes
+	- Library Updates
+2. Stack Creation (1-2 hours)
+	- See: [Creating-a-New-Stack](creating-a-new-stack.md)
+	- See: [Application-Setup](application-setup.md)
+3. Domain Switch-over (1-2 hours)
+	- See: [Domains](domains.md)
+4. Nonprofit Registration Period (30 min)
+	- Verify event settings: [Application-Setup](application-setup.md#event-settings)
+	- Prepare for minor traffic spikes [Resource-Management](resource-management.md)
+5. Donation Period (30 min)
+	- Verify payment gateway settings [Application-Setup](application-setup.md#payment-gateway-settings)
+	- Verify event settings: [Application-Setup](application-setup.md#event-settings)
+	- Prepare for high traffic spikes: [Resource-Management](resource-management.md)
+
+# Post Event Timeline
+
+1. Disable Donations (10 min)
+	- Remove payment gateway settings: [Application-Setup](application-setup.md#payment-gateway-settings)
+	- Verify event settings: [Application-Setup](application-setup.md#event-settings)
+2. Send reports to Andrew (30 min)
+	- See: [Reporting](reporting.md)
```

Comment:
This is out-dated and I don't believe necessary anymore. Allison can export this now